parachain/pallets/ethereum-light-client/src/tests.rs

```diff
@@ -719,7 +719,62 @@ fn it_should_not_allow_handle_long_range_fork_for_blocks_that_cannot_be_finalize
 }
 
 #[test]
-fn it_should_not_allow_handle_long_range_fork_for_blocks_at_or_after_current_finalized() {
+fn it_should_not_allow_force_reset_to_fork_for_blocks_at_or_after_current_finalized() {
```

Comment:
Nit: This is a very long function name. could be shortened to the following:
```suggestion
fn should_fail_reset_for_blocks_at_or_after_current_finalized() {
```